share/raja/cmake/RAJA-config.cmake.in

```diff
@@ -35,8 +35,6 @@
 include(CMakeFindDependencyMacro)
 
 if (@RAJA_NEEDS_BLT_TPLS@)
```

Comment:
This can probably be removed, and the find_dependency(Threads) moved down into camp.

Reply:
Thanks for that, for some reason I thought I had removed it 🤔 

I did a bit of cleanup otherwise, we may be able to remove some other stuff or update other versions but I kept with the ones I was sure about.